src/cpp/r/RSexp.cpp

```diff
@@ -362,16 +361,15 @@ void listEnvironment(SEXP env,
    // populate pVariables
    for (const std::string& var : vars)
    {
-      SEXP varSEXP = R_NilValue;
       // Merely calling Rf_findVar on an active binding will fire the binding.
       // Don't try to get the SEXP for the variable in this case; leave the
       // value as nil.
-      if (!isActiveBinding(var, env))
-         varSEXP = Rf_findVar(Rf_install(var.c_str()), env);
-
-      if (varSEXP != R_UnboundValue) // should never be unbound
+      if (isActiveBinding(var, env))
+         continue;
```

Comment:
Won't this keep active bindings from showing up in the environment pane altogether?

Reply:
You're right. I'll fix this up.